confidant/public/modules/resources/views/credential-details.html

```diff
@@ -38,6 +38,15 @@
       <label for="credentialEnabled">Credential Enabled</label>
       <span editable-checkbox="credential.enabled" id="credentialEnabled">{{ credential.enabled }}</span>
     </div>
+    <div class="form-group">
+      <label for="credentialGroup">Credential Group</label>
+      <span editable-select="credential.group"
+	    e-ng-options="x for x in groups( {{user.groups}} )"
+	    id="credentialGroupInput">
+	{{ credential.group || 'No group set' }}
+      </span>
+      <span ng-show="credential.id && !credential.authorized">YOU ARE NOT AUTHORIZED TO VIEW OR MODIFY THIS CREDENTIAL'S DETAILS</span>
```

Comment:
We return permission hints from individual resource level endpoints, for ACL checks; see: https://lyft.github.io/confidant/api.html#get--v1-credentials-(id)